### ISSUE TITLE
[Build] [All] v2.7.0 Error while requiring module @element-plus/nuxt: TypeError: _dayjs.default.extend is not a function

### ISSUE THREAD

**Issue by trandaison** [element-plus/element-plus#16512]
Bug Type: **`Build`**

## Environment

- Vue Version: `3.4.21`
- Element Plus Version: `2.7.0`
- Browser / OS: `macOS 14.0`
- Build Tool: `Nuxt`

## Reproduction

https://codesandbox.io/p/devbox/wonderful-elion-wm8dqg?file=%2Fpackage.json%3A8%2C33 (thanks @haozileung)

### Related Component

- `All`

### Reproduction Link

[Link]()

### Steps to reproduce

Update an existing element plus to v2.7.0

### What is Expected?

Installation should be success.

### What is actually happening?

<img width="698" alt="Screenshot 2024-04-15 at 09 44 01" src="https://github.com/element-plus/element-plus/assets/13447235/73052bf8-7976-4344-8c48-97176e049935">

Log

```
> nuxt prepare


[9:29:49 AM]  ERROR  Error while requiring module @element-plus/nuxt: TypeError: _dayjs.default.extend is not a function


 ERROR  _dayjs.default.extend is not a function                         9:29:49 AM

  at node_modules/element-plus/es/components/time-picker/src/time-picker.mjs:10:16
  at evalModule (node_modules/jiti/dist/jiti.js:1:256443)
  at jiti (node_modules/jiti/dist/jiti.js:1:254371)
  at node_modules/element-plus/es/components/time-picker/index.mjs:1:2184
  at evalModule (node_modules/jiti/dist/jiti.js:1:256443)
  at jiti (node_modules/jiti/dist/jiti.js:1:254371)
  at node_modules/element-plus/es/components/calendar/src/date-table.mjs:2:1
  at evalModule (node_modules/jiti/dist/jiti.js:1:256443)
  at jiti (node_modules/jiti/dist/jiti.js:1:254371)
  at node_modules/element-plus/es/components/calendar/src/date-table2.mjs:3:18



 ERROR  _dayjs.default.extend is not a function                         9:29:49 AM

 ELIFECYCLE  Command failed with exit code 1.
```

### Additional comments

There was an issue on element-plus-nuxt repo https://github.com/element-plus/element-plus-nuxt/issues/95, however, the lasted of `@element-plus/nuxt v1.0.8` and `element-plus v2.6.3` is still working, I think the problem should be in `element-plus 2.7.0` it self.

**Comment by btea** [comment]
Can you provide a simple repository reproduction link?

**Comment by iBobik** [comment]
Downgrade helped:
```
npm i -D @element-plus/nuxt@=1.0.8 element-plus@=2.6.3
```

**Comment by btea** [comment]
@haozileung According to the online link you provided, there is indeed an error. However, when I downloaded it and ran it locally, it seemed normal.

Could it be related to the environment? My local system is win10 and the node version is 18.19.1.

![image](https://github.com/element-plus/element-plus/assets/24516654/4f212224-cb65-45b6-8c7a-cc90f40fd0b8)

**Comment by HaManhCuong-2910** [comment]
I have same issue when build nuxt in Ubuntu, can you try it in Ubuntu environment?

**Comment by trandaison** [comment]
@HaManhCuong-2910 Chưa cần thiết dùng các tính năng của 2.7.0 thì ông hạ xuống 2.6.3 xài tạm đi đã, trong khi chờ issue được fix.

**Comment by HaManhCuong-2910** [comment]
> @HaManhCuong-2910 Chưa cần thiết dùng các tính năng của 2.7.0 thì ông hạ xuống 2.6.3 xài tạm đi đã, trong khi chờ issue được fix.
Tôi hạ xuống 2.6.3 r mà vẫn dính lỗi đấy :))

**Comment by HaManhCuong-2910** [comment]
> @HaManhCuong-2910 Chưa cần thiết dùng các tính năng của 2.7.0 thì ông hạ xuống 2.6.3 xài tạm đi đã, trong khi chờ issue được fix.

Ông fix được chưa

**Comment by trandaison** [comment]
@HaManhCuong-2910 

![Screenshot 2024-04-16 at 08 58 19](https://github.com/element-plus/element-plus/assets/13447235/df3bc132-a4ce-46b5-9be7-134366479d09)

**Comment by HaManhCuong-2910** [comment]
@trandaison Ông build dùng nuxt generate à hay là nuxt build

**Comment by damtrongan** [comment]
> Downgrade helped:
> 
> ```
> npm i -D @element-plus/nuxt@=1.0.8 element-plus@=2.6.3
> ```

I tried this workaround and applied it to Docker file. It worked for me.

**Comment by HaManhCuong-2910** [comment]
@damtrongan Which node image do you use in Dockerfile?

**Comment by damtrongan** [comment]
node:18.16-alpine3.17

**Comment by HaManhCuong-2910** [comment]
@damtrongan Are you andt02? =)))

**Comment by damtrongan** [comment]
=))) right

**Comment by HaManhCuong-2910** [comment]
@damtrongan I did see your commit "update Dockerfile" in my project =)))

**Comment by tolking** [comment]
Fixed with @element-plus/nuxt to v1.0.9.
